index.bs

```diff
@@ -660,38 +660,42 @@ Instead, you will want to stick with one of:
   : <code>bigint</code>
   :: When the possible integer values range more broadly than JavaScript numbers can accurately represent
 
-Additionally, you can combine any of the above with an extra line in your algorithm to validate
-that the number is within the expected domain-specific range, and throwing or performing other
-actions in response. (While it is very rarely appropriate to modify author input by taking it
-modulo 65536, it might be appropriate to take it modulo 360, for example.)
+Additionally, you can combine any of the above with an extra line in your
+algorithm to validate that the number is within the expected domain-specific
+range, and throwing or performing other actions in response. (While it is very
+rarely appropriate to modify author input by taking it modulo 65536, it might be appropriate to take it modulo 360, for example.)
```

Comment:
as you are already adding newlines to make the lines shorter, shouldn't you do this with the last one as well?